themes/default/theme/stencil/src/components/pulumi-pricing-calculator/pulumi-pricing-calculator.scss

```diff
@@ -0,0 +1,348 @@
+@import "../../../../src/scss/colors";
+.calculator {
+    margin-right: 36px;
+    margin-left: 36px;
```

Comment:
Would strongly prefer Tailwind classes for these.

Reply:
i based this off other stencil components where we have custom styles + i didn't see any usage of tailwind in any of those cases, so i made the assumption that it was intentional/tailwind + stencil didn't play well together.  but if that's not the case i can go back and tailwind where possible.